src/components/Reports/AdditionalSalaryRequest/Shared/AdditionalSalaryRequestContext.tsx

```diff
@@ -59,10 +69,94 @@ export const AdditionalSalaryRequestProvider: React.FC<Props> = ({
   const [sectionIndex, setSectionIndex] = useState(0);
   const selectedSection = sectionOrder[sectionIndex];
   const [isDrawerOpen, setIsDrawerOpen] = useState(true);
+
   const toggleDrawer = useCallback(() => {
     setIsDrawerOpen((prev) => !prev);
   }, []);
 
+  const createCurrencyValidation = useCallback(
+    (fieldName: string, max?: number) => {
+      let schema = amount(fieldName, t);
+      if (max) {
+        schema = schema.max(
+          max,
+          t('Exceeds ${{amount}} limit', {
```

Comment:
I believe this will show 2 dollar signs.

```suggestion
          t('Exceeds {{amount}} limit', {
```